src/main/java/thePackmaster/cards/coresetpack/Synergize.java

```diff
@@ -19,7 +27,20 @@ public Synergize() {
 
     public void use(AbstractPlayer p, AbstractMonster m) {
         blck();
-        //TODO Conditonal AOE
+        addToBot(new AbstractGameAction() {
+            @Override
+            public void update() {
+                isDone = true;
+                if (AbstractDungeon.actionManager.cardsPlayedThisCombat.size() >= 2){
+                    AbstractCard c = AbstractDungeon.actionManager.cardsPlayedThisCombat.get(AbstractDungeon.actionManager.cardsPlayedThisCombat.size() - 2);
+                    if (c instanceof AbstractPackmasterCard){
+                        if (((AbstractPackmasterCard)c).getParent().packID != getParent().packID){
```

Comment:
Again, base game cards can also be in a pack. There is a map in the main mod file cardParentMap which gives you a packID if you give it a cardID Wiz also has a method that gives you a CardPack if you give it a cardID